warehouse/upload.go

```diff
@@ -363,6 +391,8 @@ func (job *UploadJobT) run() (err error) {
 	}
 	schemaHandle := job.schemaHandle
 	schemaHandle.uploadSchema = job.upload.UploadSchema
+	// set excluded schema
+	job.generateExcludedSchema()
```

Comment:
can we move this under `!hasSchemaChanged` if block. Otherwise if its a new upload we will be generating it twice right?

Reply:
For new upload, `upload.UploadSchema` is still empty at this step. excluded schema will be empty too.
ExcludedSchema needs to be generated to get difference between `uploadSchema` and `localSchema`. `hasSchemaChanged ` is calculated based on difference between `localSchema` and `schemaInWarehouse`.